docs/source/architecture.rst

```diff
@@ -0,0 +1,34 @@
+Architecture
+============
+
+Cloudformation
+--------------
+
+Cloudformation will create the CI/CD components utilizing:
+
+    * **CodePipeline**
+    * **CodeCommit**
+    * **CodeBuild**
+    * **s3**
+    * **CloudWatch**
+    * **Cloudtrail**
+    * **GitHub**
+
+The Cloudformation template has the potential to be separately deployed for various event sources, while the only difference will be the CodeCommit repository that contains the custom configuration data for that event source.
```

Comment:
Since we now have multiple options, we should probably say "...while the only difference will be the source code repository" instead of CodeCommit.

Reply:
This has been revised, please review.

Reply:
Yeah, sounds good. Ty!